### DISCUSSION TITLE
Could you give me an example of how to create an WebSocket server using wss://

### DISCUSSION THREAD

**Discussion by matthiashack** [cesanta/mongoose#1641] [Ask questions, get answers] [answered]
I have integrate with openssl, But I don't know how to create an WebSocket server using wss:// ,could you give me some help, thank you

**Comment by scaprile** [comment, marked as answer]
Just follow [this tutorial](https://mongoose.ws/tutorials/tls/) and apply it over [this example](https://github.com/cesanta/mongoose/tree/master/examples/websocket-server).
Some of the examples, like [this one](https://github.com/cesanta/mongoose/tree/master/examples/http-restful-server). already have TLS implemented and the Makefile handles common Linux options for you; check how they handle that and you can also use those credentials for your test. Tutorials are [here](https://mongoose.ws/tutorials/).
